.circleci/config.yml

```diff
@@ -52,7 +52,26 @@ jobs:
           paths:
             - ~/.cache/yarn
 
-      - run: yarn eslint
+      - run:
+          name: eslint
+          command: yarn eslint
+
+      - run:
+          name: prettier
+          # Make the build output more useful by immediately showing full details of
+          # any style violations.
+          command: |
+            yarn prettier --write
```

Comment:
It looks like this is causing a problem: https://app.circleci.com/pipelines/github/aspiers/organice/90/workflows/1ed822e3-a87b-4132-bab8-247359674567/jobs/102/steps

`prettier` and `prettier-eslint` disagree on how to format a passage of code. I can preproduce that on my machine. The checked in code is what `prettier-eslint` generates. However, `prettier` wants it different. I propose we only use the `eslint-task`. It also accepts the `--write` flag.

Reply:
Good catch - adjusted. Let's see if the build works.

Reply:
:+1:

Reply:
Didn't work :-(  Trying to figure out why - just pushed again.

Reply:
:+1:

Reply:
Ohh, it was probably due to the merge from `master` which I only just spotted.  I'll rebase instead.

Reply:
:+1: 

I merged, because there were conflicts. And I chose the solution that `prettier-eslint` let me(;

Reply:
I've now rebased, and updated the commits/docs to reflect our preference of `prettier-eslint` over `prettier`.  Build succeeded, so I think this is ready to merge now.